great_expectations/core/batch.py

```diff
@@ -724,7 +726,7 @@ def get_batch_request_from_acceptable_arguments(
 
 def get_batch_request_dict(
     batch_request: Optional[BatchRequest] = None, validations: Optional[list] = None
-) -> dict:
+) -> Tuple[Optional[dict], Optional[list]]:
```

Comment:
It seems we took a well defined return type and make it much more ambiguous, why?

Reply:
@donaldheppner The previous return type was incorrect -- this was detected by `PyCharm`.  This change fixes it.